skbio/io/_iosources.py

```diff
@@ -29,6 +29,7 @@ def get_io_sources():
         BytesIOSource,
         BufferedIOSource,
         TextIOSource,
+        NamedTemporaryFileSource,
```

Comment:
There's I/O documentation somewhere listing the acceptable sources, can you update that too?